install/kubernetes/cilium/templates/cilium-ca.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if or
```

Comment:
nit: maybe name this file `cilium-ca-secret.yaml` to be consistent with the other files in that folder.